### ISSUE TITLE
Backwards animation.

### ISSUE THREAD

**Issue by lgarron** [cubing/twisty.js#60]
This just requires a negative direction flag, and handling inverse move transitions.

Also see cubing/alg.cubing.net#33

**Comment by pedroteosousa** [comment]
Was this fixed?

**Comment by lgarron** [comment]
Yes, in the sense that v2 supports it. Still need to make sure that all
animations work robustly and efficiently.
